tests/src/test/scala/rsc/tests/ErrorTests.scala

```diff
@@ -0,0 +1,64 @@
+package rsc.tests
+
+class ErrorTests extends RscTests {
+
+  private val errorFilesMap = errorFiles.map(path => path.getFileName.toString -> path).toMap
+
+  test("Method and field definition with No Type generates errors") {
+
+    val filename = "408.scala"
+
+    val expectedFailures = List(
+      notype(filename, "4:2..4:12", "val def <ticket408/C#x.> = \"\""),
+      notype(filename, "6:2..6:22", "protected def <ticket408/C#y().> = 42")
+    )
+
+    val failures = problemsWith(filename)
+
+    checkFailures(failures, expectedFailures)
+  }
+
+  private def problemsWith(filename: String): List[String] = {
+    rsc(errorClasspath, List(errorFilesMap(filename))).fold(problems => problems, _ => Nil)
```

Comment:
Optional: It would be a little clearer if the `.fold(...)` were indented on the next line, with a comment indicating that it is `Either.fold()`